cmd/config/init/init.go

```diff
@@ -14,23 +15,28 @@ func InitCmd() *cobra.Command {
 		Use:   "init <chain-id> <denom>",
 		Short: "Initialize a RollApp configuration on your local machine.",
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			err := verifyHubID(cmd)
-			if err != nil {
-				return err
+			interactive, _ := cmd.Flags().GetBool(FlagNames.Interactive)
+			if interactive {
+				return nil
 			}
-			err = verifyTokenSupply(cmd)
-			if err != nil {
-				return err
-			}
-			rollappID := args[0]
-			if !validateRollAppID(rollappID) {
-				return fmt.Errorf("invalid RollApp ID '%s'. %s", rollappID, getValidRollappIdMessage())
+
+			if len(args) < 2 {
+				return fmt.Errorf("invalid number of arguments. Expected 2, got %d", len(args))
 			}
+
+			//TODO: parse the config here instead of GetInitConfig in Run command
+			// cmd.SetContextValue("mydata", data)
+
 			return nil
 		},
 		Run: func(cmd *cobra.Command, args []string) {
 			initConfig, err := GetInitConfig(cmd, args)
 			utils.PrettifyErrorIfExists(err)
+
+			err = initConfig.Validate()
```

Comment:
don't we want to yell on invalid values as the user pass them in the interactive flow? so he will be able to fix them?

Reply:
did it here for now, I think it's fine for the MVP